business/data/migrate/migrate.go

```diff
@@ -18,9 +16,6 @@ import (
 var (
 	//go:embed sql/migrate.sql
 	migrateDoc string
-
-	//go:embed sql/seed.sql
-	seedDoc string
```

Comment:
Don't remove this. My project needs this for the production database and running locally. Just because you don't want it for testing doesn't make this not needed.